lib/async_hooks.js

```diff
@@ -359,9 +354,9 @@ function emitHookFactory(symbol, name) {
     } catch (e) {
       fatalError(e);
     }
-    processing_hook = false;
+    processing_hook -= 1;
```

Comment:
> I guess, but there is no way to escape fatalError so the code will never be executed.

At the moment... If at some point we'll enable a `fatalError` trap this will break. Better to be explicit about these.
Reference/depth counting only works if it's ACID